src/brasil/gov/vlibrasvideo/utils.py

```diff
@@ -149,58 +153,52 @@ def repost_news(context, event=None):
         timeout=REQUEST_TIMEOUT
     )
     logger.info('PUT - {0}'.format(params))
-    response = requests.put(**params)
-    logger.info(u'{0}: {1} - {2}'.format(
-        context.absolute_url(), response.status_code, response.reason
-    ))
-    return (response.status_code, response.reason)
+    try:
+        response = requests.put(**params)
+    except RequestException as e:  # skip on timeouts and other errors
+        logger.error(u'{0}: {1}'.format(
+            context.absolute_url(), e.message
```

Comment:
the URL of the remote server are always fixed, but the context where we got the exception is not. I think just add a text saying that it is a PUT is sufficient to know what is going on here.